tsdb/shard_test.go

```diff
@@ -1781,10 +1780,14 @@ func testFieldMaker(t *testing.T, wg *sync.WaitGroup, mf *tsdb.MeasurementFieldS
 	fields := mf.CreateFieldsIfNotExists([]byte(measurement))
 	for _, fieldName := range fieldNames {
 		if err := fields.CreateFieldIfNotExists([]byte(fieldName), influxql.Float); err != nil {
-			t.Fatalf("create field error: %v", err)
+			t.Logf("create field error: %v", err)
```

Comment:
Why not `t.Fatalf` here?

Reply:
`go vet` now reports calls to `t.FailNow()` in non-test goroutines in Go 1.18 (and `t.Fatalf()` calls `t.FailNow()`). The error it shows:
```
# github.com/influxdata/influxdb/tsdb_test
tsdb/shard_test.go:1755:3: call to (*T).Fatalf from a non-test goroutine
```